src/styles/themes/light.ts

```diff
@@ -81,15 +82,14 @@ const lightTheme = {
     skeletonLHNOut: colors.lightDefaultButtonPressed,
     QRLogo: colors.green400,
     starDefaultBG: 'rgb(254, 228, 94)',
-};
-
-lightTheme.PAGE_BACKGROUND_COLORS = {
-    [SCREENS.HOME]: lightTheme.sidebar,
-    [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
-    [SCREENS.SETTINGS.WORKSPACES]: colors.pink800,
-    [ROUTES.SETTINGS_STATUS]: colors.green700,
-    [ROUTES.I_KNOW_A_TEACHER]: colors.tangerine800,
-    [ROUTES.SETTINGS_SECURITY]: colors.ice500,
-};
+    PAGE_BACKGROUND_COLORS: {
+        [SCREENS.HOME]: colors.lightHighlightBackground,
```

Comment:
Is it safe to change from `lightTheme.sidebar` to `colors.lightHighlightBackground`?

Reply:
Yes, it's the same value, same for the dark theme